### ISSUE TITLE
Conda setup in documentation

### ISSUE THREAD

**Issue by neokd** [neokd/NeoGPT#130]
Implement support for Conda installation, ensuring seamless integration with NeoGPT. Users should be able to effortlessly set up NeoGPT using Conda, enhancing accessibility and convenience

File: docs/installation.md

**Comment by this-is-yaash** [comment]
Is the documentation implemented in `.md` or `.mdx` format?

**Comment by neokd** [comment]
In .mdx format
